wolfcrypt/src/asn.c

```diff
@@ -15427,8 +15427,126 @@ int StoreECC_DSA_Sig(byte* out, word32* outLen, mp_int* r, mp_int* s)
     return 0;
 }
 
+/* determine if leading bit is set */
+static int is_leading_bit_set(const byte* input, word32 sz)
+{
+    int i;
+    byte c = 0;
+    for (i=0; i<(int)sz; i++) {
```

Comment:
The leading zeros have been trimmed, you don't need to skip zeros anymore.

Reply:
@SparkiDev I figured it was good to leave this code in case someone else decides to utilize it in the future... But I see your point now it can just check `& 0x80`... Do you have any endianess concerns with this code?

Reply:
The byte array is assumed to be big-endian - so it will be fine.